addons/velaux/views/application-resource-detail-view.cue

```diff
@@ -9,7 +9,7 @@ parameter: {
 	kind:       string
 	apiVersion: string
 }
-response: ql.#Read & {
+response: query.#Read & {
```

Comment:
If we're changing to the new package, we also need to add `$params` to the input body, and `$returns` to get the output. You can check this as an example: https://github.com/kubevela/kubevela/blob/master/vela-templates/definitions/internal/workflowstep/collect-service-endpoints.cue